pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -409,6 +419,30 @@ func (a Adapter) execDevfile(commandsMap common.PushCommandsMap, componentExists
 	return
 }
 
+// TODO: Support Composite
+// execDevfileEvent receives a Devfile Event (PostStart, PreStop etc.) and loops through them
+// Each Devfile Command associated with the given event is retrieved, and executed in the container specified
+// in the command
+func (a Adapter) execDevfileEvent(events []string, compInfo common.ComponentInfo) error {
```

Comment:
it is better to update method parameters here, use common.Events and extract `postStart here

Reply:
I'd argue that it would be better to pass a slice of events, rather than the event struct and extracting the `postStart`, because in future implementations we could re-use this function for executing `preStop` events, for example? I wouldn't want to restrict this function to only execute `postStart` events when there are plans to implement others.

Reply:
you are right, agreed with you.